chef/data_bags/crowbar/migrate/provisioner/013_upgrade_autoyast_repos.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  unless a['suse'].empty? && a['suse']['autoyast'].empty? && a['suse']['autoyast']['repos'].empty?
+    a['suse']                           = {}
+    a['suse']['autoyast']               = {}
+    a['suse']['autoyast']['repos']      = {}
+    a['suse']['autoyast']['repos']['suse-11.3'] = a['suse']['autoyast']['repos']
+  return a, d
+end
+
+def downgrade ta, td, a, d
+  a.delete('suse')
```

Comment:
We need to move back to `a['suse']['autoyast']['repos']` if and only if there's just `a['suse']['autoyast']['repos']['suse-11.3']` under  `a['suse']['autoyast']['repos']`